### PR TITLE
Updating the YAMLs for Ingress-Citrix Addon

### PR DESCRIPTION
**Pull request by christus02**
Updating the manifest files for newer version of Citrix Ingress Controller and making it generic.

### CODE REVIEW

**Comment by christus02** [comment]
@mikesplain Could you please help in approving this PR? We have just modified the manifest files for the Citrix Ingress controller addon.

File: addons/ingress-citrix/README.md [hunk 1]
```diff
@@ -30,6 +30,7 @@ For more information on how to enable addon during cluster creation refer [Kops
 After cluster creation, you can deploy [Citrix Ingress Controller](https://github.com/citrix/citrix-k8s-ingress-controller) in Google Cloud Platform using the below command
 
 ```
+kubectl create secret generic nslogin --from-literal=username='nsroot' --from-literal=password='nsroot'
```
Comment (johngmyers) [inline addons/ingress-citrix/README.md:29]: Perhaps have the example randomly generate the password?

**Comment by johngmyers** [comment]
I like the tightening of the RBAC verbs.
You might need to push again to clear the Netlify failure.
/ok-to-test
/lgtm

File: addons/ingress-citrix/README.md (hunk 1 above)
Comment (christus02) [reply addons/ingress-citrix/README.md]: Taken care of it. Added a password placeholder.

**Comment by christus02** [comment]
> I like the tightening of the RBAC verbs.
> You might need to push again to clear the Netlify failure.
> /ok-to-test
> /lgtm

@johngmyers How do I get over the netlify failures? Pushing again did not help? Is there a way to skip it?

**Comment by rifelpet** [comment]
It looks like the netlify failures are unrelated and not flakes. I'm not sure if its an issue on netlify's side or ours but I can try to take a look soon if no one else gets to it before me. If its an issue on our side that requires a PR you'll need to rebase once that PR is merged.

**Comment by hakman** [comment]
Simplest way to rerun the tests would be to close and reopen the PR.

**Comment by christus02** [comment]
Closed and re-opened the PR in order to rerun the tests (as per the suggestion)

**Comment by hakman** [comment]
@christus02 can you rebase and force push? Your branch hasn't been updated for some time.

**Comment by christus02** [comment]
The netlify failures got resolved after rebasing. Can this PR be merged now?

**Comment by rifelpet** [comment]
Thanks!
/approve

**Pull request merged**